### PR TITLE
feat(core): upgrades lerna to 6.x.x

### PR DESCRIPTION
**Pull request by pavankjadda**
### Context
Upgrades lerna version to 6.5.1. Lerna is now maintained by https://github.com/lerna/lerna/issues/3121. Version 6.5.1 includes bug fixes and performance improvements

### Changes & Results
Upgrades Lerna to 6.5.1

### Testing
No functional changes. 

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [x] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Tested Environment

- [x] "OS:  Github codespaces
- [x] "Node version:  17
- [x] "Browser: Chrome 110
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by sedghi** [comment]
seems like the deploy is failing ... hmmm https://app.netlify.com/sites/ohif-platform-viewer/deploys/64026785e8bef10008a6fc6f

**Comment by pavankjadda** [comment]
@sedghi build is successful and all tests are passing. Please check now

File: package.json [hunk 1]
```diff
@@ -55,9 +55,11 @@
...
   },
   "dependencies": {
-    "@babel/runtime": "^7.20.13",
-    "@kitware/vtk.js": "26.5.6",
-    "core-js": "^3.2.1"
+    "@babel/runtime": "7.16.3",
+    "config-point": "^0.3.4",
+    "core-js": "^3.2.1",
+    "cornerstone-core": "2.6.0",
+    "wslink": "^0.1.8"
```
Comment (sedghi) [inline package.json:62]: I'm not sure which of these are necessary? they seem wrong to me. We don't have dependency on core-js, and you are down grading babel and adding config point ....

Comment (pavankjadda) [reply package.json:62]: Apart from these 2, do you notice anything else?

Comment (sedghi) [reply package.json:62]: we shouldn't add any dependencies that was not there before? do you need these to be added for this PR?

Comment (pavankjadda) [reply package.json:62]: Okay. I am not sure if these are added by Lerna 6.x (I hope not).  But I haven't got a chance to checkout this yet. Will let you know once I do

**Review by sedghi — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (pavankjadda) [reply package.json:62]: Sorry, didn't get a chance to work on this. Will update the PR today

Comment (pavankjadda) [reply package.json]: @sedghi Check now. Once the CI passes, you can merge this

**Comment by sedghi** [comment]
Hey, seems like you have turned off the default setting which let me push to your branch, can you please pull the latest v3-stable?

**Comment by pavankjadda** [comment]
> Hey, seems like you have turned off the default setting which let me push to your branch, can you please pull the latest v3-stable?

Done. Merged latest v3-stable changes into this branch

**Review by sedghi — APPROVED** [review]

**Comment by sedghi** [comment]
Great work, thanks for your contribution

**Pull request merged**